src/logic/FakeTouchEvent.js

```diff
@@ -53,38 +53,36 @@ export function dispatchFakeEvent(originalEvent, touch, mouseButton, type, targe
   }
 
   // trackActivePointers(type, touch, mouseButton)
+  //console.log("Sim",simulatedEvent.clientX, simulatedEvent.clientY,simulatedEvent.type,originalEvent)//,touch)
 
   target.dispatchEvent(simulatedEvent)
 }
 
 let activePointers = []
 function trackActivePointers(type, touch, mouseButton) {
   if (type === 'pointerdown') {
-    activePointers.push({ id: touch.identifier, type, mouseButton })
+    activePointers.push({ id: touch.pointerId, type, mouseButton })
   } else if (type === 'pointerup') {
-    const index = activePointers.findIndex(e => e.id === touch.identifier && e.mouseButton === mouseButton)
+    const index = activePointers.findIndex(e => e.id === touch.pointerId && e.mouseButton === mouseButton)
     if (index !== -1) {
       activePointers.splice(index, 1)
     }
   }
   if (type === 'pointerdown' || type === 'pointerup') {
-    console.log(`${type} ${touch.identifier}, button ${mouseButton}`)
+    console.log(`${type} ${touch.pointerId}, button ${mouseButton}`)
```

Comment:
Please revert all changes of this type. `pointerId` may be the underlying identifier, but the `Touch` class should abstract that away. It has the same functionality.